spec/components/blacklight/document_component_spec.rb

```diff
@@ -158,6 +158,26 @@ def call
       expect(rendered).to have_content 'embed'
     end
 
+    context 'show view with custom translation' do
+      let(:original_translations) { I18n.backend.send(:translations).deep_dup }
```

Comment:
Don't we need to do this prior to adding the new translation? Won't you need `let!` to do that?

Reply:
I'm trying to understand how this is working to restore the original traslations.  `let` blocks are called lazily, so here we're not resolving the original translations until the `after` block.  Don't we need them cached first?

Reply:
I originally tried setting it in before do like this:
before do
@original_translations = I18n.backend.send(:translations).deep_dup 
end

and then accessing it in after with @original_translations. Unfortunately rubocop started complaining and suggested using let. I moved it to the context because my assumption (which would be wrong) is it would be okay because context variables are set before before do is done.

Reply:
If you use `let!` instead of `let` then it does get the value eagerly, and it should work.